docs/Documentation/tutorial.md

```diff
@@ -1,23 +1,25 @@
+These are the instructions for running the integrated modeling system comprised of the large-scale hydrologic model **VIC** ([Liang et al, 1994](../Documentation/References.md)), a routing model based on the work of [Lohmann et al (1996)](../Documentation/References.md) and the semi-Lagrangian water temperature model, RBM ([Yearsley, 2009,2012](../Documentation/References.md)). Model development and implementation for the large-scale hydrologic model **VIC** are described in detail on the website of the University of Washington Department of Civil and Environmental Engineering's [UW Hydro|Computational Hydrology group](https://uw-hydro.github.io/code/)
 ## 1. Overview
-The **VIC-RBM** model is a coupled model that links three sub-models together: the **VIC** model; the routing model; and the **RBM** model. The **VIC** model is a hydrologic model that uses meteorological forcing data as input and simulates hydrologic variables such as runoff, evapotranspiration, and soil moisture, at each grid cell. The routing model takes the output from the **VIC** model along with flow network information as input, and output streamflow at specified locations along stream network. The **RBM** model takes both the streamflow results and the meteorological data as input and calculate stream temperature along stream network. Thus, the **VIC-RBM** model as a whole is able to simulate both streamflow and stream temperature at spatial scales determined by the basic **VIC** gridded network configuration. **VIC** gridded networks have been developed for the Continental United States (CONUS) at 1/16, 1/8, 1/4, 1/2 and 1 degree of latitude and longitude. As presently configured, the **VIC-RBM** model system simulates daily-averaged stream temperatures, only.
+The **VIC-RBM** model is a coupled model that links three sub-models together: the **VIC** model; the routing model; and the **RBM** model. The **VIC** model is a hydrologic model that uses meteorological forcing data as input and simulates hydrologic variables such as runoff, evapotranspiration, and soil moisture, at each grid cell. The routing model takes the output from the **VIC** model along with flow network information as input, and output streamflow at specified locations along stream network. The **RBM** model takes both the streamflow results and the meteorological data as input and calculate stream temperature along stream network. Thus, the **VIC-RBM** model as a whole is able to simulate both streamflow and stream temperature at spatial scales determined by the basic **VIC** gridded network configuration. **VIC** gridded networks have been developed for the Continental United States (CONUS) at 1/16, 1/8, 1/4, 1/2 and 1 degree of latitude and longitude. As presently configured, the **VIC-RBM** model system simulates daily-averaged or subdaily stream temperatures in accordance with the timestep of input.
```

Comment:
Fixed first and second. As for the last sentence, I think John means that VIC_RBM model can be set up to run at daily and subdaily timestep, which is corresponding to the input timestep.